data/text/221.txt

```diff
@@ -535,240 +535,239 @@ A sturdy Capsule that can only be\nopened with a special key.
+One of a variety of Mega Stones.\nHave Sceptile hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Blaziken hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Swampert hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Gardevoir hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Sableye hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Mawile hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Aggron hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Medicham hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Manectric hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Sharpedo hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Camerupt hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Altaria hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Banette hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Absol hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Glalie hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Salamence hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Metagross hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Latias hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Latios hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Lopunny hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Garchomp hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Lucario hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Abomasnow hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Gallade hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Audino hold it, and this stone\nwill enable it to Mega Evolve in battle.
+One of a variety of Mega Stones.\nHave Diancie hold it, and this stone\nwill enable it to Mega Evolve in battle.
```

Comment:
do these actually fit?

Reply:
should !! haven't actually tested since it's annoying to get every item by hand but im pretty sure they fit fine in pokeformatter since it's what I used to format them in the first place